airbrake/notifier.py

```diff
@@ -83,6 +70,30 @@ def __init__(self, project_id=None, api_key=None, environment=None,
                 "by passing in the arguments explicitly."
             )
 
+        environment = config.get("environment",
+                                 os.getenv('AIRBRAKE_ENVIRONMENT'))
+
+        hostname = config.get("hostname", os.getenv('HOSTNAME') or
+                              socket.gethostname())
+
+        default_base_url = os.getenv('AIRBRAKE_BASE_URL',
+                                     self.AIRBRAKE_BASE_URL_DEFAULT.strip('/'))
+        base_url = config.get("base_url", default_base_url)
+
+        self.environment = str(environment)
+        self.base_url = str(base_url)
+        self.hostname = str(hostname)
+
+        self.component = config.get("component")
+        self.action = config.get("action")
+        self.app_version = config.get("app_version")
+        self.app_url = config.get("app_url")
+        self.user_agent = config.get("user_agent")
+        self.root_directory = config.get("root_directory")
```

Comment:
Maybe? I think we'll have to be investigate this a bit. I've created https://github.com/airbrake/airbrake-python/issues/56 for this. For now we can just leave it as an option.